src/execute/index.js

```diff
@@ -49,18 +58,18 @@ const findObjectSchema = (schema, { recurse = true, depth = 1 } = {}) => {
   return undefined;
 };
 
-const parseJsonObject = ({ value, silentFail = false }) => {
+const parseJsonObjectOrArray = ({ value, silentFail = false }) => {
   try {
     const parsedValue = JSON.parse(value);
     if (typeof parsedValue === 'object') {
```

Comment:
isPlainObject(parsedValue) || Array.isArray(parsedValue)

Reply:
handles the `null` parsing, where `typeof null === 'object'` which is wrong (attribution @glowcloud)